.github/actions/fastlane/action.yaml

```diff
@@ -0,0 +1,58 @@
+name: 'fastlane'
+description: 'Runs fastlane with target'
+inputs:
+  environment:
+    required: true
+  platform:
+    reauired: true
+  target:
+    required: true
+  P12:
+    description: "P12 cert for ios sign"
+    required: false
+    default: ""
+  RELEASE_KEYSTORE:
+    description: "RELEASE_KEYSTORE for android sign"
+    required: false
+    default: ""
+  RELEASE_KEYSTORE_PASS: 
+    description: "Password for android RELEASE_KEYSTORE"
+    required: false
+    default: ""
+runs:
+  using: "composite"
+  steps:
+    - name: Set up ruby
+      if: ${{ runner.name == 'Hosted Agent' }}
```

Comment:
why do we need this IF check?

Reply:
Without it step fails on our macjenkins machine. If we plan to use that action only on selfhosted machines than we can remove it completely.

Reply:
Pls provide that explanation as inline comment

Reply:
Done